clients/admin/src/pages/dashboard/Form.tsx

```diff
@@ -0,0 +1,169 @@
+                onChange={this.handleChange('name')}
+                margin="normal"
+              />
+              <TextField
+                id="email"
+                label="Email"
+                className={classes.textField}
+                value={this.state.email}
+                onChange={this.handleChange('email')}
+                margin="normal"
+              />
+              <TextField
+                id="address"
+                label="Address"
+                className={classes.textField}
+                value={this.state.address}
+                onChange={this.handleChange('address')}
+                margin="normal"
+              />
+            </Grid>
+            <Grid item={true}>
+              <TextField
+                id="phoneNum"
+                label="Phone Number"
+                className={classes.textField}
+                value={this.state.phoneNum}
+                onChange={this.handleChange('phoneNum')}
+                margin="normal"
+              />
+              <TextField
+                id="imgPath"
+                label="Image Path"
+                className={classes.textField}
+                value={this.state.imgPath}
+                onChange={this.handleChange('imgPath')}
+                margin="normal"
+              />
+            </Grid>
+            <Grid item={true}>
+              <Button
+                variant="contained"
+                color="primary"
+                className={classes.button}
+                onClick={
+                  () => {
+                    this.props.createItem(this.state);
+                    this.props.history.goBack();
+                  }
+                }
+              >
+                Send
+                <Send className={classes.rightIcon} />
+              </Button>
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+    );
+  }
+}
+
+const styles = (theme: Theme) => ({
+  root: {
+    width: '100%',
+  },
+  boxHeader: {
+    width: '100%',
+    display: 'flex',
+    [theme.breakpoints.down('md')]: {
+      flexDirection: 'column',
+    },
+  },
+  boxHeaderTitle: {
+    padding: '5px 10px',
+    fontSize: 35,
+  },
+  fillRemainingSpace: {
+    flex: '1 1 auto',
+    [theme.breakpoints.down('md')]: {
+      display: 'none',
+    },
+  },
+    appBarSpacer: theme.mixins.toolbar,
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  textField: {
+    marginLeft: theme.spacing.unit,
+    marginRight: theme.spacing.unit,
+    width: "150px",
+  },
+  button: {
+    margin: theme.spacing.unit,
+  },
+  leftIcon: {
+    marginRight: theme.spacing.unit,
+  },
+  rightIcon: {
+    marginLeft: theme.spacing.unit,
+  },
+  formControl: {
+    marginLeft: theme.spacing.unit,
+    marginRight: theme.spacing.unit,
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  }
+});
+
+export default withRouter(withStyles(styles as any)(Form as any) as any) as any;
```

Comment:
Look at my above comment and this should fix the need for `any`. Might need to switch the order of `withRouter` and `withStyles`, but I'm not completely sure on that